runtime/doc/eval.txt

```diff
@@ -3665,7 +3668,8 @@ getftype({fname})					*getftype()*
 			getftype("/home")
 <		Note that a type such as "link" will only be returned on
 		systems that support it.  On some systems only "dir" and
-		"file" are returned.
+		"file" are returned.  On MS-Windows a symbolic link to a
+		directory returns "dir" instead of "link".
```

Comment:
Yeah, that's actually why I was asking. :) I didn't have a Windows system handy to verify if the doc updates matched neovim's behavior (yet).